cmd/ipfs-cluster-service/main.go

```diff
@@ -190,9 +190,15 @@ func main() {
 			Usage: "enable full debug logging (very verbose)",
 		},
 		cli.StringFlag{
-			Name:  "loglevel, l",
-			Value: defaultLogLevel,
-			Usage: "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+			Name:   "loglevel, l",
+			Value:  defaultLogLevel,
+			EnvVar: "LOG_LEVEL",
+			Usage:  "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+		},
+		cli.StringSliceFlag{
+			Name:   "component-loglevel, cl",
+			EnvVar: "COMP_LOG_LEVEL",
+			Usage:  "set separate loglevel for individual cluster components",
```

Comment:
We can use a single flag for everything right?

"debug"  means all cluster
"debug,swarm2:info" means all facilities to debug and swarm2 to info
"raft:error", means all facilities with default logging except raft.

Reply:
I have pushed changes to do everything together.

But I find the previous way better. Now if someone uses "swarms=info" or "swam2-info" it would say that it is not a valid log level, which is less informative than saying `use the formate identifier:loglevel`